invokeai/backend/model_manager/probe.py

```diff
@@ -132,12 +132,14 @@ def probe(
 
         format_type = ModelFormat.Diffusers if model_path.is_dir() else ModelFormat.Checkpoint
         model_info = None
-        model_type = None
-        if format_type is ModelFormat.Diffusers:
-            model_type = cls.get_model_type_from_folder(model_path)
-        else:
-            model_type = cls.get_model_type_from_checkpoint(model_path)
-        format_type = ModelFormat.ONNX if model_type == ModelType.ONNX else format_type
+        model_type = fields["type"] if "type" in fields else None
+        model_type = ModelType(model_type) if isinstance(model_type, str) else model_type
```

Comment:
You don't really need to do this check. `ModelType(model_type)` will return a ModelType regardless of whether model_type is a string or a ModelType already.

Reply:
Interesting, I only added it cause I was getting errors for some reason

Reply:
oh I'm not getting the error now weird. Maybe I was misinterpreting another error

Reply:
Oh the check is in case the line above returns None, but yeah it could just check for not None instead